ebpfsvc/eBPFSvc.vcxproj

```diff
@@ -130,10 +131,10 @@
       <SDLCheck>true</SDLCheck>
       <PreprocessorDefinitions>_DEBUG;EBPFSVC_EXPORTS;_WINDOWS;_USRDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
-      <PrecompiledHeader>Use</PrecompiledHeader>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
```

Comment:
If this project doesn't use a precompiled header, then delete the pch.h file.

Reply:
Changed pch.h to svc_common.h and removed pch.cpp file